demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -119,9 +129,11 @@ def add_from_version_to_yml(self, file_path=None, yml_content=None, save_yml=Tru
                 LATEST_SUPPORTED_VERSION) > parse_version(yml_content.get('fromversion', '0.0.0')):
             yml_content['fromversion'] = LATEST_SUPPORTED_VERSION
 
+            self.fix_script_in_unified_yml(yml_content)
+
             if save_yml:
-                with open(file_path, 'w') as f:
-                    yaml.dump(yml_content, f)
+                with open(file_path, mode='w', encoding='utf-8') as f:
+                    YAML().dump(yml_content, f)
```

Comment:
why do we need this change

Reply:
According to @yaakovi this is a better way to write a yaml since it works alongside the FoldedScalarString function that prevents the change in the script.